partiql-parser/src/main/antlr/PartiQL.g4

```diff
@@ -555,7 +555,7 @@ mathOp00
     ;
 
 mathOp01
-    : lhs=mathOp01 op=(PLUS|MINUS) rhs=mathOp02
```

Comment:
Are we sure on these operator precedence rules for `&`? Is this based off another DBMS?

Reply:
This behavior follows [SQL server](https://learn.microsoft.com/en-us/sql/t-sql/language-elements/operator-precedence-transact-sql?view=sql-server-ver16). 

It is worth to note that the bitwise operator is not in the SQL spec, so the operator precedence is different across multiple DBMSs.

Reply:
Could we add TODO referring a spec issue for bitwise operators precedence?

Reply:
Okay I see. The current precedent seems reasonable to me. For reference, it looks like [postgresql](https://www.postgresql.org/docs/16/sql-syntax-lexical.html#SQL-PRECEDENCE) has it around the same place ("(any other operator)" which follows addition and subtraction).

Reply:
I agree. The SQL server's implementation is rather not universal in this situation. I change the implementation to so that plus/minus have higher precedence.